desc/compute/_curve.py

```diff
@@ -168,16 +168,21 @@ def _Z_Curve(params, transforms, profiles, data, **kwargs):
     description="Position vector along curve",
     dim=3,
     params=["r_n", "center", "normal", "rotmat", "shift"],
-    transforms={
-        "r": [[0, 0, 0]],
-    },
+    transforms={"r": [[0, 0, 0]]},
     profiles=[],
     coordinates="s",
     data=["s"],
     parameterization="desc.geometry.curve.FourierPlanarCurve",
     basis="{'rpz', 'xyz'}: Basis for returned vectors, Default 'rpz'",
+    basis_in="{'rpz', 'xyz'}: Basis for input params vectors, Default 'xyz'",
 )
 def _x_FourierPlanarCurve(params, transforms, profiles, data, **kwargs):
+    if kwargs.get("basis_in", "xyz").lower() == "rpz":
```

Comment:
How does this compute function know about kwargs["basis_in"]? Where is that happening in the code? I'm trying to replicate it in the spline branch but it's not working :( i just pushed it if either of you would like to try and help -- it's probably something simple I'm missing

Reply:
I overwrote the class's `compute` method so that it always passes in this kwarg:
https://github.com/PlasmaControl/DESC/blob/dd/rpz_curve/desc/geometry/curve.py#L736

Reply:
Ohh okay thanks daniel!